### DISCUSSION TITLE
Wazuh api web login

### DISCUSSION THREAD

**Discussion by DragurNaut** [Security-Onion-Solutions/securityonion#10180] [Unsupported Versions]
Hello,

How do you login to the Wazuh api on port 55000? I have tried the Wazuh official defaults as well as my SO console and soc credentials. I have also tried various traditional "default" credentials (admin:admin, admin:password, etc...).

**Comment by cm-ops** [comment]
What are you trying to do?

**Comment by DragurNaut** [comment]
I would like to edit the syslog input filter and output

**Comment by cm-ops** [comment]
Have you tried editing the` ossec.conf` on the server?
